### PR TITLE
Refactoring to hooks

### PR DESCRIPTION
**Pull request by anagudelogu**
# In this Milestone I:

- [x] Refactored Button component to be functional and use React hooks.
- [x] Refactored Calculator component to be functional.

Feel free to give any optional feedback and suggestions about the project and clean code in general, I'm always trying to improve and write professional code. 🤓

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
 import PropTypes from 'prop-types';
 
-export default class Button extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Button(props) {
+  const { handleClick, classN, value } = props;
```
Comment (thecodechaser) [inline src/components/Button.js]: - [x] Please use the arrow function-based components to follow ES6 syntax.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,96 +1,80 @@
...
-      calc: {},
-    };
-    this.operationsRight = ['÷', 'x', '-', '+', '='];
-    this.operationsCenter = ['AC', '+/-', '%'];
-    this.handleClick = this.handleClick.bind(this);
-  }
+export default function Calculator() {
+  const OPERATIONS_RIGHT = ['÷', 'x', '-', '+', '='];
+  const OPERATIONS_CENTER = ['AC', '+/-', '%'];
+  const DIGITS = [
```
Comment (thecodechaser) [inline src/components/Calculator.js]: - [x] Please use the arrow function-based components to follow ES6 syntax.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @anagudelogu👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (anagudelogu) [reply src/components/Calculator.js]: @thecodechaser sure will do! thank you.

**Review by gemmen29 — APPROVED** [review]
Hi @anagudelogu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**